src/App.tsx

```diff
@@ -2,6 +2,7 @@ import { BrowserRouter, Routes, Route } from "react-router-dom";
 import HomePage from "./components/HomePage";
 import PlaceholderPage from "./components/PlaceholderPage";
 import PageLayout from "./components/PageLayout";
+import Quiz from "./components/Quiz";
```

Comment:
Ah and maybe it would be good to call this component `QuizPage` so it's clear it's one of the top level page components

Reply:
Done